apps/demos/testing/widgets/accordion/Accordion.test.js

```diff
@@ -0,0 +1,59 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+const ITEM_CONTENT_CLASS = 'dx-item-content';
+const CHECKBOX_ICON_CLASS = 'dx-checkbox-icon';
+
+fixture('Accordion.Overview')
+  .page('http://localhost:8080/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 600];
+  });
+
+runManualTest('Accordion', 'Overview', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('Custom Overview Appearance', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t
+      .click($(`.${ITEM_CONTENT_CLASS}`).nth(2))
```

Comment:
Why do u click a content, not just an item?
Now u have non-intuitive indices because of this